### DISCUSSION TITLE
Does not start up automatically on MacOS Ventura

### DISCUSSION THREAD

**Discussion by Fregf** [borgbase/vorta#1721] [FAQ]
Just set up an Apple Macbook Pro ARM with latest Ventura and latest Vorta installed via homebrew. In the misc tab, Automatically Start Vorta at login is enabled. Vorta is shown in System Preferences Login Items. However, when loggin in as that user, the Vorta icon does not appear in the top bar, and according to Activity Monitor it is not running. I can manually start Vorta from /Applications and then it works fine. But of course, I want this to happen automatically, so that backups will be run according to the configured schedule without manual user intervention.

Any idea how I could debug why it's not starting at login?

**Comment by real-yfprojects** [comment]
The described behaviour indicates Vorta exits with an Error when run through autostart. Is something like a system log where you can find the output from the Vorta process that was started by autostart? Do Vorta's log files contain any entries for the autostart? 
A similar or even the same was reported in #1064 already. However we failed to reproduce.
